src/test/java/com/amazon/ion/bytecode/bin11/bytearray/PrimitiveDecoderTest.kt

```diff
@@ -160,6 +161,18 @@ class PrimitiveDecoderTest {
         assertEquals(expected, actual)
     }
 
+    @ParameterizedTest
+    @MethodSource(FIXED_INT_8_CASES, FIXED_INT_16_CASES, FIXED_INT_24_CASES, FIXED_INT_32_CASES, FIXED_INT_64_CASES)
+    @CsvSource(
+        " 9223372036854775808, 00000000 00000000 00000000 00000000 00000000 00000000 00000000 10000000 00000000",
+        "-9223372036854775809, 11111111 11111111 11111111 11111111 11111111 11111111 11111111 01111111 11111111",
```

Comment:
We can add in some over-padded values too.
```suggestion
        " 9223372036854775808, 00000000 00000000 00000000 00000000 00000000 00000000 00000000 10000000 00000000",
        "-9223372036854775809, 11111111 11111111 11111111 11111111 11111111 11111111 11111111 01111111 11111111",
        "                   1, 00000001 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000",
        "                  -1, 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111",
```